edit-post/components/layout/index.js

```diff
@@ -94,6 +95,7 @@ function Layout( {
 			}
 			<Popover.Slot />
 			<PluginArea />
+			<GuideTip.Slot />
```

Comment:
I moved away from `Popover` as there’s some differences in appearance and behaviour. We probably will want to extract common functionality into a shared component e.g. `PopoverBase`. I’m wary of introducing this abstraction while we’re experimenting with the NUX UI, though.

Reply:
I just made some adjustments to the Popover component and I think it's great in resolving all the positioning/size issues. I'd love to know what are those difference to see if we can bake them into the Popover component. From the screenshots it looks very similar to the popover component. I believe what's missing is the support of the "center" position in the yAxis. We only support it in the `xAxis`.

Reply:
I’ll check out your adjustments. When I last looked, the differences were:

- Blue dot instead of an arrow
- Should appear to the left or right of the anchor point instead of above or below
- Clicking outside shouldn’t dismiss

Reply:
> Blue dot instead of an arrow

The blue dot should be the anchor (outside the popover)

> Should appear to the left or right of the anchor point instead of above or below

Yes, that's what's missing the "center" support for the yAxis. I believe we can add it.

> Clicking outside shouldn’t dismiss

This should work, maybe a bug somewhere

> Must appear above other popovers

Popovers take classNames, I believe we can adjust the z-index for nux popovers

Reply:
I'm adding a `noArrow` prop to this one that could also help https://github.com/WordPress/gutenberg/pull/6911#issuecomment-391325040